ml-agents/mlagents/trainers/sac/optimizer_torch.py

```diff
@@ -199,13 +199,9 @@ def __init__(self, policy: TorchPolicy, trainer_params: TrainerSettings):
             1e-10,
             self.trainer_settings.max_steps,
         )
-        self.policy_optimizer = torch.optim.Adam(
-            policy_params, lr=hyperparameters.learning_rate
-        )
-        self.value_optimizer = torch.optim.Adam(
-            value_params, lr=hyperparameters.learning_rate
-        )
```

Comment:
adding the learning rate as argument to Adam is misleading because these two optimizers use a learning rate schedule that overrides this value anyway.